docs/reference/reference-training/training/api-pytorch-trainer-reference.rst

```diff
@@ -0,0 +1,120 @@
+###############################
+ PyTorch Trainer API Reference
+###############################
+
++----------------------------------------------------+
+| User Guide                                         |
++====================================================+
+| :doc:`/training/apis-howto/api-pytorch-trainer-ug` |
++----------------------------------------------------+
+
+.. code::
+
+   determined.pytorch.init(
```

Comment:
why the handwritten reference doc, instead of using normal sphinx autodocs?

This will look odd surrounded by the rest of our reference docs which all use that technique.